docs/reference/components/sinks/datadog_logs.cue

```diff
@@ -3,6 +3,13 @@ package metadata
 components: sinks: datadog_logs: {
 	title: "Datadog Logs"
 
+	description: """
+		Sends data to the [Datadog logs service](\(urls.datadog_logs)). The Datadog platform can leverage specifics fields (namely `ddtags`, `ddsource` and `service`) to further categorize
```

Comment:
s/the Datadog logs service/Datadog Log Management

s/leverage/use

s/specifics/specific

comma after `ddsource`